lib/types.d.ts

```diff
@@ -13,9 +13,7 @@ declare global {
   type JSXFragment = ASTNode;
   type JSXSpreadAttribute = ASTNode;
 
-  interface Context extends eslint.SourceCode {
-    getFirstTokens(node: estree.Node | ASTNode, options?: eslint.SourceCode.CursorWithCountOptions): eslint.AST.Token[];
-  }
+  type Context = eslint.Rule.RuleContext
```

Comment:
why this change?

Reply:
That Context type definition is a duct tape definition. It lacks many methods.

Reply:
ah, so this one includes getFirstTokens, awesome